shared/clients/email/email.go

```diff
@@ -20,25 +21,31 @@ type SMTP struct {
 	auth     smtp.Auth
 }
 
-type TemplateData struct {
+type InviteTemplateData struct {
 	Name  string
 	Title string
 }
 
+type ResetPasswordTemplateData struct {
```

Comment:
This template is for `Forget Password` but the variable name is `ResetPasswordTemplateData`, WDYT about changing it to something related to forget password?